config.go

```diff
@@ -91,6 +91,10 @@ type ReceiverConfig struct {
 	Description       string                 `yaml:"description" json:"description"`
 	WontFixResolution string                 `yaml:"wont_fix_resolution" json:"wont_fix_resolution"`
 	Fields            map[string]interface{} `yaml:"fields" json:"fields"`
+	Components        []string               `yaml:"components" json:"components"`
+
+	// Label copy settings
+	CopyPromLabels bool `yaml:"copy_prom_labels" json:"copy_prom_labels"`
```

Comment:
"AddGroupLabels", maybe? "Group labels" is more specific, otherwise users may expect all labels to be applied.

Reply:
Good point. Just done.